### PR TITLE
New step description and input

### PR DESCRIPTION
**Pull request by BanyikAnna**
Please check the descriptive part and the input texts as well.

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -37,46 +55,45 @@ inputs:
...
       title: "Private key password"
-      summary: ""
-      description: |
-        If key password equals to keystore password, you can leave it empty.
-        Otherwise specify the private key password.
+      summary: "Password you added to the private key."
+      description:
       is_sensitive: true
-  - bundletool_version: "1.0.0"
+  - bundletool_version: "0.15.0"
```
Comment (adborbas) [inline step.yml:84]: Please use "1.0.0" for the bundletool_version as we have updated to latest version on master.

File: step.yml [hunk 2]
```diff
@@ -37,46 +55,45 @@ inputs:
       is_required: true
   - keystore_url: $BITRISEIO_ANDROID_KEYSTORE_URL
     opts:
-      title: "Keystore url"
-      summary: ""
+      title: "Keystore URL"
+      summary: "The keystore file's URL which is generated when you upload the file to the Code Signing tab."
       description: |-
         TO BE FILLED
```
Comment (adborbas) [inline step.yml:61]: `TO BE FILLED` is left here.

File: step.yml [hunk 3]
```diff
@@ -27,8 +45,8 @@ toolkit:
 inputs:
   - aab_path:
     opts:
-      title: "The path of the AAB."
-      summary: Summary. No more than 2-3 sentences.
+      title: "Android App Bundle path"
+      summary: "Android App Bundle file (`.aab`) path"
       description: |
         Description of this input.
```
Comment (adborbas) [inline step.yml:51]: Description is not updated.

File: step.yml [hunk 4]
```diff
@@ -37,46 +55,45 @@ inputs:
...
+      description:
       is_sensitive: true
-  - bundletool_version: "1.0.0"
+  - bundletool_version: "0.15.0"
     opts:
-      title: Version of bundletool to use.
-      summary: |
+      title: "Bundletool version"
+      summary: "The latest Bundletool version is set automatically."
+      description: "If you wish to set a specific version other than the latest, add it here based on [Bundletool's official release](https://github.com/google/bundletool/releases) page."
```
Comment (adborbas) [inline step.yml:88]: Please use the multiline format with `|` like at line 5.

File: step.yml [hunk 5]
```diff
@@ -27,57 +45,52 @@ toolkit:
...
       is_sensitive: true
   - bundletool_version: "1.0.0"
     opts:
-      title: Version of bundletool to use.
-      summary: |
+      title: "Bundletool version"
+      summary: "The latest Bundletool version is set automatically."
+      description:  |
+      "If you wish to set a specific version other than the latest, add it here based on [Bundletool's official release](https://github.com/google/bundletool/releases) page."
         Versions are available here: https://github.com/google/bundletool/releases
```
Comment (adborbas) [inline step.yml:87]: This line is outside of the `description` tag. Ending " is one line above.

File: step.yml [hunk 6]
```diff
@@ -27,57 +45,50 @@ toolkit:
...
       is_required: true
   - keystore_url: $BITRISEIO_ANDROID_KEYSTORE_URL
     opts:
-      title: "Keystore url"
-      summary: ""
-      description: |-
-        TO BE FILLED
+      title: "Keystore URL"
+      summary: "The keystore file's URL which is generated when you upload the file to the Code Signing tab."
+      description: ""
```
Comment (lpusok) [inline step.yml:58]: The description should include the same summary if not more info. Did not enfoce this everywhere previously. The workflow editor UI displays the description when expanding a step input (not really consistently.) Or would reccomend modifying the workflow editor to fall back to the summary in an expanded view.

**Pull request merged**